core/src/main/java/dev/hooon/show/domain/entity/seat/Seat.java

```diff
@@ -29,6 +33,8 @@
 @NoArgsConstructor(access = PROTECTED)
 public class Seat extends TimeBaseEntity {
 
+    private static final String SEAT = "seat";
```

Comment:
이거는 어디에 쓰이나요?!

Reply:
```suggestion
        Assert.notNull(show, getNotNullMessage(SEAT, "show"));
        Assert.notNull(seatGrade, getNotNullMessage(SEAT, "seatGrade"));
        Assert.hasText(sector, getNotEmptyPostfix(SEAT, "sector"));
        Assert.hasText(row, getNotEmptyPostfix(SEAT, "row"));
        Assert.notNull(showDate, getNotNullMessage(SEAT, "showDate"));
        Assert.notNull(startTime, getNotNullMessage(SEAT, "startTime"));
        Assert.notNull(seatStatus, getNotNullMessage(SEAT, "seatStatus"));
```
여기에 사용되며 템플릿을 보면 제가 어떤 도메인에서 발생한 예외인지 파악하려고 prefix 로 도메인 이름을 받도록 설정했습니다!